src/Sodaq_dataflash.h

```diff
@@ -26,6 +26,8 @@
 
 #define DF_AT45DB081D   1
 #define DF_AT45DB161D   2
+// Adding AT45DB041D:
+#define DF_AT45DB161D   3
 
 #define DF_VARIANT      DF_AT45DB161D
```

Comment:
I meant something like this:

    #ifndef DF_VARIANT
    #define DF_VARIANT DF_AT45DB161D
    #endif